File: Erik_Vincent/Assignment1/Assignment1/Model/QLS/AST/Style/Widget/SpinBox.cs
using Assignment1.Model.QL.AST;

namespace Assignment1.Model.QLS.AST.Style.Widget
{
    public class SpinBox : ASTNode, IWidget
    {
        public SpinBox(int lineNumber)
        {
            _lineNumber = lineNumber;
        }

        public void Accept(IStyleVisitor visitor) => visitor.Visit(this);
    }
}

File: GunOli/src/main/java/QLS/AST/Widgets/SpinBox.java
package QLS.AST.Widgets;

import QLS.Analysis.WidgetVisitorInterface;

public class SpinBox extends Widget {


    public SpinBox(int line) {
        super(WidgetType.SpinBox, line);
    }



    @Override
    public <T> T accept(WidgetVisitorInterface<T> visitor) {
        return visitor.visit(this);
    }
}
